ext/standard/filestat.c

```diff
@@ -771,13 +771,13 @@ PHPAPI void php_stat(const char *filename, size_t filename_length, int type, zva
 		"size", "atime", "mtime", "ctime", "blksize", "blocks"
 	};
 	const char *local;
-	php_stream_wrapper *wrapper;
 
 	if (!filename_length) {
 		RETURN_FALSE;
 	}
 
-	if ((wrapper = php_stream_locate_url_wrapper(filename, &local, 0)) == &php_plain_files_wrapper && php_check_open_basedir(local)) {
+	php_stream_wrapper *wrapper = php_stream_locate_url_wrapper(filename, &local, 0);
```

Comment:
This will not work as `PHP-7.2` is not subject to C99, the decl needs to be at top block level (I think it was only `PHP-7.4` and forward we opted to this, @nikic?)

Reply:
no problem, will roll out this change

Reply:
Thank you :)